pkg/model/build_reason.go

```diff
@@ -29,17 +33,23 @@ func (r BuildReason) IsCrashOnly() bool {
 }
 
 var translations = map[BuildReason]string{
-	BuildReasonFlagChangedFiles: "Changed Files",
-	BuildReasonFlagConfig:       "Config Changed",
-	BuildReasonFlagCrash:        "Pod Crashed, Lost live_update Changes",
-	BuildReasonFlagInit:         "Initial Build",
+	BuildReasonFlagChangedFiles:   "Changed Files",
+	BuildReasonFlagConfig:         "Config Changed",
+	BuildReasonFlagCrash:          "Pod Crashed, Lost live_update Changes",
+	BuildReasonFlagInit:           "Initial Build",
+	BuildReasonFlagTriggerWeb:     "Button Pressed",
+	BuildReasonFlagTriggerCLI:     "Commandline Trigger",
```

Comment:
maybe "CLI trigger"? (at minimum "command-line" should be hyphenated)

Reply:
but also for this and "button pressed" it's really that we _forced_ an update (if it's releasing a pending update on a manual resource, that update shows up as "files changed"). Not sure I have good words here, but "forced update via web" / "forced update via CLI"?

Reply:
changed it to CLI Trigger

It feels important that it leads with a noun, rather than a subjectless verb? "Forced Update" puts the emphasis on the wrong place, the part the user doesn't really care about

Reply:
sure, I buy that.